tests/viet_kwik_e_mart/domains/store_info/get_store_hours/testtrain123.txt

```diff
@@ -0,0 +1,310 @@
+When does the {Elm Street|store_name} store close?
```

Comment:
Remove all "testtrain*.txt" files from the vietnamese folders

Reply:
Done!